x-pack/plugins/observability_solution/observability_onboarding/e2e/playwright/stateful/pom/pages/onboarding_home.page.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Page } from '@playwright/test';
+
+export class OnboardingHomePage {
+  page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  private readonly useCaseKubernetes = () =>
+    this.page.locator(
+      'xpath=//div[@data-test-subj="observabilityOnboardingUseCaseCard-kubernetes"]//input[@type="radio"]'
```

Comment:
You know, I just realized that our Team's POC is the only place I know of in the repo that has a playwright-data-test-subj integration.

Here [it is](https://github.com/elastic/kibana/blob/693a16bf7279efc2da0ca54cc5f63a37b5879495/packages/kbn-scout/src/playwright/fixtures/types/test_scope.ts#L61).

In this case the kbn/scout package does not override playwrights test id attr, but this test subj functionality is exposed.  

Maybe this is useful for you all?

\cc @dmlemeshko

Reply:
Those are good points about using `xpath`, I left them as they were originally written in the oblt-playwright repo, but it 100% makes sense to just use test id helper and narrow down using Locator's methods, I'll re-write those selector